backend/src/services/user.go

```diff
@@ -19,3 +23,15 @@ type UserService struct {
 func (u *UserService) GetAllUsers() ([]models.User, error) {
 	return transactions.GetAllUsers(u.DB)
 }
+
+// Delete user with a specific id
+func (u *UserService) DeleteUser(id string) (error) {
+	idAsInt, err := strconv.Atoi(id)
```

Comment:
Definitely something to be extracted into some validation function with how commonly this will be done

Reply:
Let's leverage the `utilities.ValidateID` here